js/base/common/.eslintrc

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
I've decided to isolate these 2 files in a folder (common) for two reasons:
- we need to do some linting (to avoid breaking the transpiling) and the eslintrc in base/ is not compatible with the rules we want
- does not quite fit in the js/* folder because we need slightly different linting here since this is not a class, unlike the derivated files.